NetCoreRestApi/ServiceLayer/DataTransferObjects/CategoryDTO.cs

```diff
@@ -1,9 +1,8 @@
 ﻿
 namespace ServiceLayer.DataTransferObjects
 {
-    public class CategoryDTO
-    {
-        public int Id { get; set; }
+    public class CategoryDTO : BaseDto
```

Comment:
Create ItemDto, ItemModel, ItemEntity that contains Name and Description properties